client/blocks/importer/wordpress/import-everything/confirm.tsx

```diff
@@ -48,35 +48,36 @@ export const Confirm: React.FunctionComponent< Props > = ( props ) => {
 	const [ showUpgradePlanScreen, setShowUpgradePlanScreen ] = useState( false );
 	const sourceSiteSlug = sourceSite?.slug ?? '';
 
-	/**
-	 ↓ Effects
-	 */
-	useEffect( () => {
-		const skipCta = getQueryArg( window.location.href, 'skipCta' );
-		recordTracksEvent( 'calypso_site_importer_migration_confirmation' );
-		if ( isTargetSitePlanCompatible && isMigrateFromWp && skipCta ) {
-			startImportCta();
-		}
-	}, [] );
-
 	/**
 	 ↓ Functions
 	 */
-	function showConfirmDialogOrStartImport() {
+	const showConfirmDialogOrStartImport = useCallback( () => {
```

Comment:
I think this file is a part of old flow and not using anymore at the moment. Can't triggering this during my test.